crates/runtime-secrets/src/stores/scheduler_rpc.rs

```diff
@@ -0,0 +1,57 @@
+use crate::{AnyErrorResult, SecretStore};
+use async_trait::async_trait;
+use prost::{Message, bytes};
+use runtime_proto::{ExecutorExpandSecretRequest, ExecutorExpandSecretResponse};
+use secrecy::SecretString;
+
+/// Used by cluster mode to resolve secrets declared in the scheduler
+/// via flight RPC
```

Comment:
This is a new secrets store used exclusively on executors to expand secrets. It purposefully does not cache them locally to avoid situations like: AWS secret is rotated, a new value can be polled from API, but isn't because the local cache is stale.